vscode_extension/src/sorbetStatusProvider.ts

```diff
@@ -125,6 +125,12 @@ export class SorbetStatusProvider implements Disposable {
    */
   public async startSorbet(): Promise<void> {
     if (this.isStarting) {
+      this.context.log.trace("Ignoring start request, already starting.");
+      return;
+    }
+
+    if (!this.context.configuration.activeLspConfig) {
+      this.context.log.info("Ignoring start request, no active configuration.");
```

Comment:
Should we add something here like, "... If you want to use Sorbet in this project, please add a `sorbet/config` file" or whatever instructions people would have to follow to get it to start?

Reply:
Following up as https://github.com/sorbet/sorbet/pull/7152